go.mod

```diff
@@ -113,6 +113,7 @@ require (
 	github.com/kr/text v0.2.0
 	github.com/nsf/jsondiff v0.0.0-20210926074059-1e845ec5d249
 	github.com/openark/golib v0.0.0-20210531070646-355f37940af8
+	go.uber.org/multierr v1.8.0
```

Comment:
Let's also update to `1.9.0` here since that's the latest?

Reply:
done